pkg/cmd/cli/cmd/process.go

```diff
@@ -89,7 +110,7 @@ Examples:
 			// If 'parameters' flag is set it does not do processing but only print
 			// the template parameters to console for inspection.
 			if cmdutil.GetFlagBool(cmd, "parameters") == true {
-				err = printer.PrintObj(templateObj, out)
+				err = printTemplateParameters(templateObj, out)
```

Comment:
Ah, you should be passing templateObj.Parameters here, and then keep printTemplateParameters in describe/printer.go (and register template parameters as its own type).